src/rev_ai/apiclient.py

```diff
@@ -272,47 +289,57 @@ def get_transcript_object(self, id_):
             raise ValueError('id_ must be provided')
 
         response = self._make_http_request(
-                       "GET",
-                       urljoin(self.base_url, 'jobs/{}/transcript'.format(id_)),
-                       headers={'Accept': self.rev_json_content_type}
+            "GET",
+            urljoin(self.base_url, 'jobs/{}/transcript'.format(id_)),
+            headers={'Accept': self.rev_json_content_type}
         )
 
         return Transcript.from_json(response.json())
 
-    def get_captions(self, id_):
+    def get_captions(self, id_, content_type=CaptionType.SRT, channel_id=None):
         """Get the captions output of a specific job and return it as plain text
 
         :param id_: id of job to be requested
+        :param content_type: caption type which should be returned. Defaults to SRT
+        :param channel_id: id of speaker channel to be captioned, only matters for multichannel jobs
         :returns: caption data as text
         :raises: HTTPError
         """
         if not id_:
             raise ValueError('id_ must be provided')
+        query = ""
+        if channel_id:
+            query = "?speaker_channel={}".format(channel_id)
 
         response = self._make_http_request(
-                       "GET",
-                       urljoin(self.base_url, 'jobs/{}/captions'.format(id_)),
-                       headers={'Accept': self.rev_captions_content_type}
+            "GET",
+            urljoin(self.base_url, 'jobs/{0}/captions{1}'.format(id_, query)),
+            headers={'Accept': content_type.value}
         )
 
         return response.text
 
-    def get_captions_as_stream(self, id_):
+    def get_captions_as_stream(self, id_, content_type=CaptionType.SRT, channel_id=None):
         """Get the captions output of a specific job and return it as a plain text stream
 
         :param id_: id of job to be requested
+        :param content_type: caption type which should be returned. Defaults to SRT
+        :param channel_id: id of speaker channel to be captioned, only matters for multichannel jobs
         :returns: requests.models.Response HTTP response which can be used to stream
             the payload of the response
         :raises: HTTPError
         """
         if not id_:
             raise ValueError('id_ must be provided')
+        query = ""
+        if channel_id:
+            query = "?speaker_channel={}".format(channel_id)
```

Comment:
same question about extracting shared logic into helpers

Reply:
Moving this to a shared helper seems unnecessary. It's not much duplicate code and the helper function is not useful on its own.

Reply:
I worry about future developers not implementing changes in both locations.